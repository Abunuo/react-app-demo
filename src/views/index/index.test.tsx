/*
 * @Date: 2022-01-12 17:49:11
 * @Description: Index test
 */
import React from "react"
import renderer from 'react-test-renderer';
import MainIndex from './index';

it('MainIndex show Dom', () => {
  const component = renderer.create(
    <MainIndex></MainIndex>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});