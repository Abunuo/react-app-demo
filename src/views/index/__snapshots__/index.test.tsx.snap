// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainIndex show Dom 1`] = `
<div>
  <h1>
    index
  </h1>
</div>
`;
